routes/admin/admin-view.route.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function (req, res, next) {
```

Comment:
The name could just be admin, it would be more consistent with the other routes.

The route should be done in a consistent way with the other ones. see the order.route.js file as an example.